shm/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.10)
+set(PROJECT_NAME SHM)
+project(${PROJECT_NAME})
+FILE(GLOB SRC_FILES "*.cpp")
+list(REMOVE_ITEM SRC_FILES main.cpp test.cpp)
+
+
+add_library(${PROJECT_NAME}-lib STATIC ${SRC_FILES})
```

Comment:
I don't know that SCR_FILES has main.cpp or not? Once you define SCR_FILES by file, later by list, so in this situation what happens? :)

Reply:
SRC_FILES created by `files` adds all *.cpp files by default. So I have to remove them from SRC_FILES using this command. Still better than adding stuff one by one :D

Reply:
nice, good to know ;)